dev/reference/performance/statistics.md

```diff
@@ -69,6 +91,29 @@ ANALYZE TABLE TableName PARTITION PartitionNameList [IndexNameList] [WITH NUM BU
 
 在执行分析索引列任务的时候，`tidb_index_serial_scan_concurrency` 可以用于控制一次读取的 Region 数量，其默认值是 1。
 
+### 查看 ANALYZE 状态
+
+在执行 `ANALYZE` 时，可以通过 SQL 语句来查看当前 `ANALYZE` 的状态。
+
+语法：
+
+```sql
+SHOW ANALYZE STATUS [ShowLikeOrWhere]
+> 该语句会输出 `ANALYZE` 的状态，可以通过使用 `ShowLikeOrWhere` 来筛选需要的信息。
```

Comment:
```suggestion
-- 该语句会输出 `ANALYZE` 的状态，可以通过使用 `ShowLikeOrWhere` 来筛选需要的信息。
```

SQL 中使用 -- 或者 # 来作为行注释。另外我觉得把这个注释写到 code block 外面会更好

Reply:
不需要吧，这个文件其他地方都是这种格式。

Reply:
@yikeke Please help review the code blocks in this PR.

Reply:
We have a [new guide](https://docs.google.com/document/d/1tgVFl8Rol-gxV0aLgDymCivQg_7PKpkMhR1mxo2cVic/edit?usp=sharing) about the code block format. In most cases, we should put the code comment outside of the code block because it's easier for users to copy the codes in the block. 

And in our documents, we normally use "--" or "#" to add a code comment. See [Code Comment Style](https://github.com/pingcap/community/blob/master/code-comment-style.md) for details.

Reply:
I'll update the code blocks for you so you can see how our new code block rules look like. @lamxTyler

Reply:
Updated. PTAL. @lamxTyler @lilin90 @zz-jason